packages/tests/jest.config.ts

```diff
@@ -2,14 +2,22 @@
 // This product includes software developed at Datadog (https://www.datadoghq.com/).
 // Copyright 2019-Present Datadog, Inc.
 
-module.exports = {
+import { createJsWithTsPreset, type JestConfigWithTsJest } from 'ts-jest';
+
+const presetConfig = createJsWithTsPreset({
+    // ...options
```

Comment:
is that something we want to change later ?

Reply:
Good catch, I forgot to remove this after my copy pasting from their website 🤦
I'll remove it.